control-plane/grpc/src/context.rs

```diff
@@ -161,7 +161,7 @@ impl<C: Clone> Client<C> {
     {
         let context = Context::new(options);
         let endpoint = context.endpoint(uri);
-        let channel = endpoint.connect_lazy().unwrap();
+        let channel = endpoint.connect_lazy();
```

Comment:
Question: Did the function signature change? Is there any change in behaviour?

Reply:
It did yes, but no change in behaviour, it never used to return any error anyway.

Reply:
Okay